composer.json

```
@@ -11,8 +11,8 @@
         }
     ],
     "require": {
-        "php": "^7.2",
-        "symfony/framework-bundle": "^4.4 || ^5.0",
+        "php": ">=7.2.5",
+        "symfony/framework-bundle": "^4.4 || >=5.0",
```

```suggestion
        "php": "^7.2.5||^8.0",
        "symfony/framework-bundle": "^4.4 || ^5.0",
```

[breaking changes](https://jubianchi.github.io/semver-check/#/constraint/%3E%3D5.0)